src/cache.ts

```diff
@@ -3,6 +3,9 @@ import { TemplateElementsConnection } from './utils/generated/graphql'
 
 const cache = new InMemoryCache({
     typePolicies: {
+      Application: {
+        keyFields: ['serial']
```

Comment:
who does this need to be done ?

Reply:
This was done while I was trying to fix what was causing the cache management to not work properly. The problem happening was that after an application was fetched (onLoadApplication), regardless being present in the cache, the App was trying to fetch from the server again for the same application because we are using the `serial` to find it. The cache doesn't have a way to check for other fields in an entity that is already in the cache just by its identifier (aka `keyField`). Since we are using the `serial` to query for applications at all places it makes more sense to use it as the identifier in the cache. And now the Apollo cache is correctly getting the application from cache the 2nd time.